avocado/plugins/run.py

```diff
@@ -224,4 +240,13 @@ def run(self, args):
             result_dispatcher.map_method('render',
                                          job_instance.result,
                                          job_instance)
+        # Export the serialized avocado_variants
+        if args.dump_variants is not None:
+            try:
+                with open(args.dump_variants, 'w') as variants_file:
+                    json.dump(args.avocado_variants.dump(), variants_file)
```

Comment:
If you chose to keep the `--dump-variants` for `run` consider copying the `jobdata/variants.json` instead of serializing it again. It's quite costly operation compare to pure copy.